### PR TITLE
Update RoPE initialization in mixtral.py

### PR DESCRIPTION
**Pull request by ivanfioravanti**
Fix for issue #465  
The RoPE initialization in the mixtral.py file has been updated. Previously, RoPE was being instantiated directly, but now it has been switched to be instantiated through the nn module.

### CODE REVIEW

File: llms/mixtral/mixtral.py [hunk 1]
```diff
@@ -76,7 +56,7 @@ def __init__(self, args: ModelArgs):
         self.wk = nn.Linear(args.dim, args.n_kv_heads * args.head_dim, bias=False)
         self.wv = nn.Linear(args.dim, args.n_kv_heads * args.head_dim, bias=False)
         self.wo = nn.Linear(args.n_heads * args.head_dim, args.dim, bias=False)
-        self.rope = RoPE(args.head_dim, traditional=True)
+        self.rope = nn.RoPE(args.head_dim, traditional=True)
```
Comment (angeloskath) [inline llms/mixtral/mixtral.py:59]: In order for this to match the above we should probably pass the base parameter to RoPE. ie `nn.RoPE(args.head_dim, base=1000000, traditional=True)`.

**Review by angeloskath — CHANGES_REQUESTED** [review]
Thanks so much for doing that! Did you by any chance test that the generations still make sense?

Left a small comment for a forgotten base argument.

File: llms/mixtral/mixtral.py (hunk 1 above)
Comment (ivanfioravanti) [reply llms/mixtral/mixtral.py]: It was working well even with base Mistral value 🤔

Comment (angeloskath) [reply llms/mixtral/mixtral.py]: Hm interesting. Let me go search which one should be there and then I 'll merge this. Thanks!

Comment (ivanfioravanti) [reply llms/mixtral/mixtral.py]: 1000000 is the right one, but please use #470 from @jbrodriguez

**Comment by angeloskath** [comment]
Thanks @ivanfioravanti for the fix and for giving it up to @jbrodriguez :-)

**Pull request closed without merging**